### PR TITLE
Refactor server integration tests

### PR DESCRIPTION
**Pull request by brfrn169**
Please take a look!

### CODE REVIEW

File: server/src/integration-test/java/com/scalar/db/server/ServerEnv.java [hunk 1]
```diff
@@ -35,7 +35,7 @@ public final class ServerEnv {
 
   public ServerEnv() {}
 
-  public static Properties getServerProperties1(String testName) {
+  public static Properties getServer1Properties(String testName) {
```
Comment (brfrn169) [inline server/src/integration-test/java/com/scalar/db/server/ServerEnv.java:38]: Renamed these methods in `ServerEnv`.

File: server/src/integration-test/java/com/scalar/db/server/ConsensusCommitAdminRepairTableIntegrationTestWithDistributedTransactionAdminService.java [hunk 2]
```diff
@@ -87,6 +90,14 @@ public void repairTableAndCoordinatorTable_ForTruncatedMetadataTable_ShouldRepai
     super.repairTableAndCoordinatorTable_ForTruncatedMetadataTable_ShouldRepairProperly();
   }
 
+  /** This test is disabled if {@link #isExternalServerUsed()} return true */
+  @Override
+  @Test
+  @DisabledIf("isExternalServerUsed")
+  public void repairTable_ForCorruptedMetadataTable_ShouldRepairProperly() throws Exception {
```
Comment (brfrn169) [inline server/src/integration-test/java/com/scalar/db/server/ConsensusCommitAdminRepairTableIntegrationTestWithDistributedTransactionAdminService.java:97]: @Torch3333 I think we also need to disable this method if `isExternalServerUsed` is true, right?

File: server/src/integration-test/java/com/scalar/db/server/DistributedStorageAdminServiceRepairTableIntegrationTest.java [hunk 3]
```diff
@@ -60,6 +65,14 @@ public void repairTable_ForTruncatedMetadataTable_ShouldRepairProperly() throws
     super.repairTable_ForTruncatedMetadataTable_ShouldRepairProperly();
   }
 
+  /** This test is disabled if {@link #isExternalServerUsed()} return true */
+  @Override
+  @Test
+  @DisabledIf("isExternalServerUsed")
+  public void repairTable_ForCorruptedMetadataTable_ShouldRepairProperly() throws Exception {
```
Comment (brfrn169) [inline server/src/integration-test/java/com/scalar/db/server/DistributedStorageAdminServiceRepairTableIntegrationTest.java:72]: @Torch3333 Ditto. I think we also need to disable this method if isExternalServerUsed is true, right?

File: server/src/integration-test/java/com/scalar/db/server/SchemaLoaderIntegrationTestWithScalarDbServer.java [hunk 4]
```diff
@@ -38,4 +48,30 @@ public void afterAll() throws ExecutionException, IOException {
...
+
+  /** This test is disabled if {@link #isExternalServerUsed()} return true */
+  @Override
+  @Test
+  @DisabledIf("isExternalServerUsed")
+  public void
+      createTableThenDropMetadataTableThenRepairTablesWithCoordinator_ShouldExecuteProperly()
+          throws Exception {
+    super.createTableThenDropMetadataTableThenRepairTablesWithCoordinator_ShouldExecuteProperly();
+  }
```
Comment (brfrn169) [inline server/src/integration-test/java/com/scalar/db/server/SchemaLoaderIntegrationTestWithScalarDbServer.java:69]: @Torch3333 Ditto. I think we also need to disable these methods if isExternalServerUsed is true, right?

File: server/src/integration-test/java/com/scalar/db/server/ConsensusCommitAdminRepairTableIntegrationTestWithDistributedTransactionAdminService.java (hunk 2 above)
Comment (Torch3333) [reply server/src/integration-test/java/com/scalar/db/server/ConsensusCommitAdminRepairTableIntegrationTestWithDistributedTransactionAdminService.java:97]: Yes, thank you for fixing this.

File: server/src/integration-test/java/com/scalar/db/server/DistributedStorageAdminServiceRepairTableIntegrationTest.java (hunk 3 above)
Comment (Torch3333) [reply server/src/integration-test/java/com/scalar/db/server/DistributedStorageAdminServiceRepairTableIntegrationTest.java:72]: Yes, thank you for fixing this.

File: server/src/integration-test/java/com/scalar/db/server/SchemaLoaderIntegrationTestWithScalarDbServer.java (hunk 4 above)
Comment (Torch3333) [reply server/src/integration-test/java/com/scalar/db/server/SchemaLoaderIntegrationTestWithScalarDbServer.java:69]: Yes, thank you for fixing this.

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Review by komamitsu — APPROVED** [review]
LGTM, thanks!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**